File: EE1083/Jee 2018 through Python/14.c
#include <stdio.h>
#include <math.h>

int main(void)
{
float x,y,z;
int i;

for (i = 0; i<21; i++)
{
x = i;
z = x - 4*(sqrt(x));
y = x - 8*(sqrt(x)) + 12;

if(i<10)
{
printf("%f\n", y);
}

else
{
printf("%f\n", z);
}

}
return 0;
}

File: EE1083/Jee 2018 through Python/14.py
import numpy as np
import matplotlib.pyplot as plt

plt.axis([0,20,-10,20])

x2 = np.linspace(0,9,100)
x1 = np.linspace(9,20,100)

y = x1 -4*(np.sqrt(x1))
z = x2 + 12 - 8*(np.sqrt(x2))

plt.plot(x1,y, label = '$f(x) = x - 4*(x)^0.5$')
plt.plot(x2,z, label = '$f(x) = x - 8*(x)^0.5 + 12$')

sol = np.zeros((2,1))
sol[0] = 9
sol[1] = -3

A = sol[0]
B = sol[1]


plt.plot(A,B,'o')
for xy in zip(A,B):
	plt.annotate('(%s, %s)'%xy,xy=xy,xytext=(30,0), textcoords='offset points')
	
plt.grid()
plt.legend(loc='best' ,prop={'size':11})
plt.xlabel('$x$')
plt.ylabel('$f(x)$')
plt.show()
